CONTRIBUTING.md

```diff
@@ -0,0 +1,54 @@
+# Contributing
+
+## Setting up a dev environment
+
+Though voila-gallery is deployed as a [TLJH](https://tljh.jupyter.org)
+plugin, you can develop it locally without needing TLJH. You just need
+[docker](https://docker.com) and Python 3.6+ installed.
+
+1. Create a virtual environment to work in, with your favorite
+   Python virtual environment tool. With Python's built in `venv`
+   module, you can make a virtual environment with:
+
+   ```bash
+   python3 -m venv .
+   source bin/activate
+   ```
+
+2. Install development requirements
+
+   ```bash
+   pip install -r dev-requirements.txt
+   ```
+2. Do a dev install of the `tljh_voila_gallery` package.
+
+   ```bash
+   pip install -e tljh-voila-gallery
+   ```
+
+3. Build all the images
+
+   ```bash
+   python3 -m tljh_voila_gallery.build_images
+   ```
+
+   This requires docker be available and usable from your current
+   user without root. If you need to use `sudo` before your `docker`
+   calls, you should use `sudo $(which python3)` rather than just
+   `python3`
+
+4. Start JupyterHub with the testing config
```

Comment:
We also need `configurable-http-proxy` before starting JupyterHub. And node as well unless it's available via other channels?

Reply:
Ah, true! I already had them installed I think so didn't explicitly add them. I'll add pointers to both...

Reply:
Added!

Reply:
Thanks!